app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ListsController < ApplicationController
+
+  # users must be signed in before any lists_controller method
+  before_action :authenticate_user!
+
+  def index
+    @lists = List.all
+  end
+
+  def show
+    @list = List.find(params[:id])
```

Comment:
> Could it be something to do with the fact the user_id field in the db isn't being populated and is required by devise for the current_user method?

It does sound like something along those lines, perhaps the list.user_id value isn't populated in the database, or the `current_user` doesn't yet have a list. It'd be worth taking a look at the data in the development database via `rails c`. Once in there, look at all the created lists, and see if any have null `user_id` values. This is a guess, so I might be on the wrong track here. We can take a closer look in the next call if needed.